app/web/features/donations/locales/ja.json

```diff
@@ -22,7 +22,5 @@
     "donations_text": "",
     "donations_text2": "",
     "benefactor_contact": "",
-    "benefactor_email": "benefactor@couchers.org",
-    "donations_recurrence_explainer": "",
-    "donations_value": "{{val, currency}}"
+    "donations_recurrence_explainer": ""
```

Comment:
Incidently, these should probably not be here, right? Otherwise it won't fall back

Reply:
(I mean the blank strings)

Reply:
It would fallback fine since we have `returnEmptyString` configured to false in `next-i18next` for situation like these, as Weblate seems to output untranslated string as empty sometimes...